resources/launcher-dev.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='UTF-8'/>
+    <link rel='stylesheet' type='text/css' href='../dist/launcher.css'>
```

Comment:
This depends on the existence of the file `dist/launcher.css` which is not automatically built by `npm run dev` anymore. I think this is fixed by also getting the CSS from the dev server:
```suggestion
    <link rel='stylesheet' type='text/css' href='http://localhost:5004/launcher.css'>
```

Reply:
Good call, done that. And how fantastically useful that GitHub suggested change feature is!